### PR TITLE
chore(lint): remove JSHint

### LINKED ISSUE #159: Remove JSHint
Remove the JSHint hooks still left in code in the fxa-js-client once the ESLint integration is complete.

### PR DESCRIPTION
**Pull request by TDA**
Fixes #159
Remove the evil JSHint from code base
All Hail ESLint
@pdehaan 
@vladikoff

### CODE REVIEW

**Comment by TDA** [comment]
@pdehaan Sensei, am I ready?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Patches should be submitted as pull requests (PR).
 
 Before submitting a PR:
 - Your code must run and pass all the automated tests before you submit your PR for review. "Work in progress" pull requests are allowed to be submitted, but should be clearly labeled as such and should not be merged until all tests pass and the code has been reviews. 
-  - Run `grunt jshint` to make sure your code passes linting.
+  - Run `grunt eslint` to make sure your code passes linting.
```
Comment (vladikoff) [inline CONTRIBUTING.md]: should this be `grunt lint`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ When submitting a PR:
 - You agree to license your code under the project's open source license ([MPL 2.0](/LICENSE)).
 - Base your branch off the current `master` (see below for an example workflow).
 - Add both your code and new tests if relevant.
-- Run `grunt jshint` and `npm test` to make sure your code passes linting and tests.
+- Run `grunt eslint` and `npm test` to make sure your code passes linting and tests.
```
Comment (vladikoff) [inline CONTRIBUTING.md]: same as comment above

File: CONTRIBUTING.md (hunk 1 above)
Comment (TDA) [inline CONTRIBUTING.md]: This repo doesnt do grunt lint Vlad :)

Comment (TDA) [inline CONTRIBUTING.md]: nvm, i will change it.

Comment (vladikoff) [inline CONTRIBUTING.md]: :scream: 

code below:
image

Comment (TDA) [inline CONTRIBUTING.md]: I must have gotten it confused with some other repo :| Shame on me!

**Comment by TDA** [comment]
@vladikoff Now? "Am I ready Sensei?" ( you are supposed to say, "You are ready, my son")

**Pull request merged**